metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -124,6 +124,9 @@ class SalesforceConfig(DatasetSourceConfigMixin):
         default=dict(),
         description='Regex patterns for tables/schemas to describe domain_key domain key (domain_key can be any string like "sales".) There can be multiple domain keys specified.',
     )
+    sf_version: Optional[str] = Field(
```

Comment:
Suggestion: Rename this to `api_version` so that it is inline with other salesforce specific parameters such as `access_token` and `consumer_key`. We don't typically add prefixes to integration-specific parameters and the config is already integration-specific

Reply:
it's a good idea, renamed